### PR TITLE
Divide temporal resoultion in resolutionValue and resolutionUnit #207

### LINKED ISSUE #207: The "timeseries" - "resolution" should be split into two fields like "spatial" - "resoultion"
## Description of the issue

The "timeseries" - "resolution" should be split into two fields like "spatial" - "resoultion"

## Ideas of solution

                    "resolutionValue": "30",
                    "resolutionUnit": "s",

## Workflow checklist
- [ ] I am aware of the workflow in [CONTRIBUTING.md](https://github.com/OpenEnergyPlatform/oemetadata/blob/develop/CONTRIBUTING.md)

### PR DESCRIPTION
**Pull request by Ludee**
## Summary of the discussion

The "timeseries" - "resolution" should be split into two fields like "spatial" - "resoultion".
```JSON
"resolutionValue": "30",
"resolutionUnit": "s",
```

### Updated
- Separate `timeseries - resolution` into `resolutionValue` and `resolutionUnit` [(#213)](https://github.com/OpenEnergyPlatform/oemetadata/pull/213)

### Automation
Closes #207

### PR-Assignee
- [x] 🐙 Follow the workflow in [CONTRIBUTING.md](https://github.com/OpenEnergyPlatform/oemetadata/blob/develop/CONTRIBUTING.md)
- [x] 📝 Update the [CHANGELOG.md](https://github.com/OpenEnergyPlatform/oemetadata/blob/develop/CHANGELOG.md)
- [x] 📙 Update the documentation
- [x] 🐙 Assign a reviewer to the PR

### Reviewer
- [x] 🐙 Follow the [Reviewer Guidelines](https://github.com/OpenEnergyPlatform/oemetadata/blob/develop/CONTRIBUTING.md#40-let-someone-else-review-your-pr)
- [x] 🐙 Provided feedback and show sufficient appreciation for the work done

### CODE REVIEW

File: metadata/v20/v20/example.json [hunk 1]
```diff
@@ -75,7 +75,8 @@
                     {
                         "start": "2020-01-01T00:00:00+00:00",
                         "end": "2020-01-01T23:59:30+00:00",
-                        "resolution": "30 s",
+                        "resolutionValue": "15",
```
Comment (jh-RLI) [inline metadata/v20/v20/example.json:78]: I think this is better than what we had, but there also seem to be specific codes that express resolution in seconds, ... .  Maybe some users want to use them. Nonetheless, this way is very meaningful and human-readable.

Comment (jh-RLI) [reply metadata/v20/v20/example.json:78]: More a comment

**Review by jh-RLI — APPROVED** [review]
Looks fine to me

**Comment by jh-RLI** [comment]
A the file called metadata named "mapping...." seems a bit confusion to me. Is it really a mapping`?

**Comment by Ludee** [comment]
Yes, I added the mapping to DCAT-AP to the documentation. This will be migrated to OMI and linked!

**Comment by jh-RLI** [comment]
Ah, okay it is an actual mapping then that makes sense :) This can be merged.

**Pull request merged**